src/kusto/azext_kusto/generated/_params.py

```diff
@@ -39,126 +39,130 @@ def load_arguments(self, _):
+                   'al tenants.')
         c.argument('optimized_autoscale', action=AddOptimizedAutoscale, nargs='+', help='Optimized auto scale definitio'
-                   'n. Expect value: KEY1=VALUE1 KEY2=VALUE2 ... , available KEYs are: version, is-enabled, minimum, ma'
-                   'ximum.')
+                   'n.')
         c.argument('enable_disk_encryption', arg_type=get_three_state_flag(), help='A boolean value that indicates if t'
                    'he cluster\'s disks are encrypted.')
         c.argument('enable_streaming_ingest', arg_type=get_three_state_flag(), help='A boolean value that indicates if '
                    'the streaming ingest is enabled.')
         c.argument('virtual_network_configuration', action=AddVirtualNetworkConfiguration, nargs='+', help='Virtual net'
-                   'work definition. Expect value: KEY1=VALUE1 KEY2=VALUE2 ... , available KEYs are: subnet-id, engine-'
-                   'public-ip-id, data-management-public-ip-id.')
+                   'work definition.')
         c.argument('key_vault_properties', action=AddKeyVaultProperties, nargs='+', help='KeyVault properties for the c'
-                   'luster encryption. Expect value: KEY1=VALUE1 KEY2=VALUE2 ... , available KEYs are: key-name, key-ve'
-                   'rsion, key-vault-uri.')
+                   'luster encryption.')
         c.argument('enable_purge', arg_type=get_three_state_flag(), help='A boolean value that indicates if the purge o'
                    'perations are enabled.')
-        c.argument('language_extensions_value', action=AddLanguageExtensionsValue, nargs='+', help='The list of languag'
-                   'e extensions. Expect value: language-extension-name=xx.')
+        c.argument('enable_double_encryption', arg_type=get_three_state_flag(), help='A boolean value that indicates if'
+                   ' double encryption is enabled.')
         c.argument('identity_type', arg_type=get_enum_type(['None', 'SystemAssigned']), help='The identity type.')
-        c.argument('identity_user_assigned_identities', arg_type=CLIArgumentType(options_list=['--identity-user-assigned-identities'], help='The list of user identities associated with the Kusto cluster. The user identity dictionary key references will be ARM resource ids in the form: \'/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}\'. Expected value: json-string/@json-file.'))
+        c.argument('identity_user_assigned_identities', arg_type=CLIArgumentType(options_list=['--identity-user-assigne''d-identities'], help='The list of user identities associated with the Kusto cluster. The user ident''ity dictionary key references will be ARM resource ids in the form: \'/subscriptions/{subscriptionI''d}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{i''dentityName}\'. Expected value: json-string/@json-file.'))

```

Comment:
Is this on purpose? As parameter's name, `d-identities` is quite weird.

Reply:
Resolved in manual folder